app/store/networkDrawer.ts

```diff
@@ -0,0 +1,51 @@
+import { CTransactionSegWit } from '@defichain/jellyfish-transaction'
+import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+export interface Transaction {
```

Comment:
```suggestion
export interface TransactionJob {
```

For not to confused with jellyfish existing Transaction interface

Reply:
I'll rename it OceanTransaction since it will be called OceanInterface now